smoke-tests/Makefile

```diff
@@ -17,6 +17,20 @@ atlanta/cobblinc-flex.gtfs.zip:
 atlanta/gtfs.zip: atlanta/cobblinc.gtfs.zip atlanta/cobblinc-flex.gtfs.zip
 	${CURL} https://itsmarta.com/google_transit_feed/google_transit.zip -o $@
 
+houston/osm.pbf:
+	# this file can be re-hosted on IBI provided infrastructure
+	${CURL} https://leonard.io/ibi/houston-2022-08-26.osm.pbf -o $@
```

Comment:
You can probably use our OSM extract server for this! Check out http://osm.ibi-transit.com which tells you how to make the request. 

If it fails we might have to whitelist the ci. Let me know how it goes!

Reply:
Cool, thanks for the tip. Is there an endpoint in the API that lets me use WGS84 coordinates rather than decimal degrees?

Reply:
Just double checked looking at the source (https://github.com/conveyal/osm-lib) and it doesn't seem like it... You'd have to do the conversion locally

Reply:
Ah, I figured it out. I had already used decimal degrees but swapped lat/long: http://osm.ibi-transit.com/vex/29.65285233056443,-95.48492431640624,29.857914952075095,-95.22193908691406.pbf

It's quite slow though, at 400kb/second. Is this normal?

Reply:
Actually, it's cached by Github Actions so doesn't matter much.